src/main/java/org/jclouds/vcloud/director/v1_5/compute/functions/ImageForVAppTemplate.java

```diff
@@ -97,10 +97,10 @@ public Image apply(QueryResultVAppTemplateRecord from) {
           try {
               builder.location(findLocationForResource.apply(reference));
           } catch (NoSuchElementException e) {
-              logger.warn("The image doesn't have a location metadata attached and cannot be prioritized", e);
+              logger.debug("The image %s doesn't have location metadata attached. %s", from, e.toString());
```

Comment:
For future reference, don't need to call `toString()`. The logger will call toString on each arg if and only if the log level means it needs to be written. So you could have just passed `e`. But fine as-is.